marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/step/saveStep.sjs

```diff
@@ -84,6 +84,9 @@ else {
   else if("merging" === stepDefinitionType){
     stepDefinitionName = "default-merging";
   }
+  else if("custom" === stepDefinitionType){
+    stepDefinitionName = "default-custom";
```

Comment:
The entire "else-if" block has to be removed. Custom steps can have any step definition name

Reply:
Fixed